### ISSUE TITLE
Web component: Card group - With tab shift tab key we are able to reach to the card in card but there is no focus indicator 

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6693]
## Detailed description

Issue:
With tab shift tab key we are able to reach to the card in card but there is no focus indicator

Expected:
Provide thick and strong visual focus.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/a28e3b17-05f9-492c-a36e-eca12f5774ef)
